packages/practice/fecrash-cafe/cypress/e2e/app.cy.js

```diff
@@ -0,0 +1,14 @@
+describe('FECrash 카페', () => {
+  beforeEach(() => {
+    cy.visit({
+      url: 'http://localhost:5500/packages/practice/fecrash-cafe/index.html',
```

Comment:
```suggestion
      url: './index.html',
```

아래 이슈 확인해보니 live server 사용하지 않고도 가능한 것 같습니다!

https://github.com/cypress-io/cypress/issues/4450